recurly/base_client.py

```diff
@@ -64,4 +64,4 @@ def _make_request(self, method, path, body, params):
     def _interpolate_path(self, path, *args):
         """Encodes components and interpolates path"""
 
-        return path % tuple(map(urllib.parse.quote, args))
+        return path % tuple(map(lambda arg: urllib.parse.quote(arg, safe=""), args))
```

Comment:
Do we need to worry about `+` as well? Should we use [`quote-plus`](https://docs.python.org/3.6/library/urllib.parse.html#urllib.parse.quote_plus) instead of `quote`?

I'm not sure offhand if account codes can have spaces in them. It doesn't mention that on our docs page https://docs.recurly.com/docs/accounts#section-account-code

Reply:
I don't think that `quote-plus` is necessary. The `quote` method will encode spaces to `%20` where `quote-plus` will change them to `+` (unescaped). The plus sign is encoded by both methods:

```python
>>> urllib.parse.quote("+ /", safe="")
'%2B%20%2F'
>>> urllib.parse.quote_plus("+ /")
'%2B+%2F'
```

Reply:
Alright, just making sure. This looks like the right solution